sdk/communication/azure-communication-callingserver/src/test/java/com/azure/communication/callingserver/CallConnectionAsyncLiveTests.java

```diff
@@ -27,6 +28,14 @@
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
     public void runCreatePlayCancelHangupScenarioAsync(HttpClient httpClient) {
+
+        // This test requires human intervention to record and
+        // will not function in live mode.
+        if (this.getTestMode() == TestMode.LIVE) {
+            System.out.println("Warning: Test is skipped, does not support live mode.");
+            return;
+        }
+
```

Comment:
Consider using a pattern similar to this.

@DisabledIfEnvironmentVariable(
        named = "COMMUNICATION_SKIP_INT_PHONENUMBERS_TEST",
        matches = "(?i)(true)")

c.f sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java#L90

Reply:
this pattern is applied here and other related places